index.html

```diff
@@ -14,259 +14,315 @@
+  </div>
+</section>
+<section class="about">
+  <div class="about-me__wrapper">
+    <div class="about-me">
+      <h3 class="about-me__heading">About me</h3>
+      <p class="about-me__paragraph">Hello I’m a software developer! I can help you 
+          build a product , feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="about-me__button">
+          <a href="#" class="about-me__button--link">Get My Resume</a>
+      </div>
+      
+    </div>
+    <div class="about-bg"></div>
+  </div>
+  <div class="language-container"> 
+    <div class="about-languages">
+      <h3 class="about-languages__heading">
+        <img src="./img/Rectangle55.svg" alt="" class="about__icon">
+          Languages
+      </h3>
+      <ul class="about-languages__list">
+        <li class="about-languages__list--item">Javascript</li>
+        <li class="about-languages__list--item">Ruby</li>
+        <li class="about-languages__list--item">Html</li>
+        <li class="about-languages__list--item">Css</li>
+      </ul>
+    </div>
+      <div class="about-frameworks">
+        <h3 class="about-frameworks__heading">
+          <img src="./img/Rectangle56.svg" alt="" class="about__icon">
+          Frameworks
+        </h3>
+        <ul class="about-frameworks__list">
+          <li class="about-frameworks__list--item">Bootstrap</li>
+        <li class="about-frameworks__list--item">Ruby on Rails</li>
+        <li class="about-frameworks__list--item">RSPec</li>
+        <li class="about-frameworks__list--item">CapyBara</li>
+        <li class="about-frameworks__list--item">Selenium</li>
+        </ul>
+      </div>
+      <div class="about-skills">
+        <h3 class="about-skills__heading">
+          <img src="./img/Rectangle57.svg" alt="" class="about__icon">
+          Skills
+        </h3>
+        <ul class="about-skills__list">
+          <li class="about-skills__list--item">Codekit</li>
+          <li class="about-skills__list--item">Github</li>
+          <li class="about-skills__list--item">Codepen</li>
+          <li class="about-skills__list--item">Gitlab</li>
+          <li class="about-skills__list--item">Terminal</li>
+        </ul>
+      </div>
+    
+  </div>
+</section>
+<footer class="contact-form">
+  <form action="https://formspree.io/f/mgedzjjn" method="POST" class="form desktop">
+    <h3 class="form__heading">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+    <ul class="form__list form__list--big">
+        <li class="form__list--item fullname">
+          <input type="text" name="first-name" maxlength="30" placeholder="First name" required class="first-name">
+
+          <input type="text" name="last-name" maxlength="30" placeholder="Last name" required class="last-name">
```

Comment:
- [ ] I know it's a bit confusing but as per requirements let's make sure we only have one input which is `full name`
![image](https://user-images.githubusercontent.com/87689547/159916799-ba968796-133f-409d-8dfe-eaaad60477dd.png)

Reply:
Thank you, I thought I had to implement this, I spend almost an hour trying to get this to work thank you.